guides/doc-Upgrading_and_Updating/topics/updating_satellite_server_to_next_minor_version.adoc

```diff
@@ -1,17 +1,7 @@
 [[updating_satellite_server_to_next_minor_version]]
 
 
-= Updating {ProjectServer}
-
-.Prerequisites
-
-* Ensure that you have synchronized {ProjectServer} repositories for {Project}, {SmartProxy}, and {project-client-name}.
-* Ensure each external {SmartProxy} and Content Host can be updated by promoting the updated repositories to all relevant Content Views.
-
-[WARNING]
-If you customize configuration files, manually or use a tool such as Hiera, these customizations are overwritten when the installation script runs during upgrading or updating.
-You can use the `--noop` option with the {foreman-installer} script to test for changes.
-For more information, see the Red Hat Knowledgebase solution https://access.redhat.com/solutions/3351311[How to use the noop option to check for changes in {Project} config files during an upgrade].
+= Updating {ProjectServer} =
```

Comment:
Hmm look here -> https://partha.fedorapeople.org/docs/build/Upgrading_and_Updating/index-katello.html#updating_satellite_server_to_next_minor_version looks valid to me there ?  

It may be incorrect according to ascii doc guide. Going to check and update.

Reply:
I think you are right going to apply

Reply:
Cool. Glad it works.  We don't do that in general, just use the = 

THANK YOU SO MUCH for previewing.